UKHomeOffice/nmsw-ui
# NMSW-3 Setup react app

# Ticket

NMSW-3

----

## AC

Setup React app with webpack and associated and common packages

----

## To test

- run app locally
- go to localhost:3000
- > you should see a h1 of 'Basic setup'

----

## Notes

This includes:

- [x] Webpack
- [x] babel
- [x] React
- [x] eslint
- [x] dotenv
- [x] sass
- [x] prop types
- [x] index.html
- [x] index.js
- [x] App.jsx
- [x] package.json scripts
- [x] gitignore
